eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Mgmt/src/Providers/ResourceClientProvider.cs

```diff
@@ -52,8 +52,8 @@ public ResourceClientProvider(InputClient inputClient)
             SpecName = resourceModel.Name;
 
             // We should be able to assume that all operations in the resource client are for the same resource
-            var requestPath = new RequestPath(inputClient.Operations.First().Path);
-            _resourceOperations = inputClient.Operations;
+            var requestPath = new RequestPath(inputClient.Methods.First().Operation.Path);
+            _resourceServiceMethods = inputClient.Methods.ToList();
```

Comment:
Why ToList?

Reply:
Oops, I had a Select to operations before, forgot to remove ToList after removing Select.
Removed now.